src/main/java/org/jabref/gui/help/AboutDialogViewModel.java

```diff
@@ -29,6 +28,7 @@
     private static final String LICENSE_URL = "https://github.com/JabRef/jabref/blob/master/LICENSE.md";
     private final String changelogUrl;
     private final String versionInfo;
+    private final ReadOnlyStringWrapper environmentInfo = new ReadOnlyStringWrapper();
```

Comment:
The `environmentInfo` is not needed (or set), right?

Reply:
I did not change that. environmentInfo is use line 37 in AboutDialog.fxml